tools/scrumcheck/ui/assets/app.js

```
@@ -0,0 +1,1074 @@
+        });
+        panels.forEach((panel) => {
+          panel.classList.toggle('active', panel.id === 'tab-' + tabName);
+        });
+      }
+      buttons.forEach((btn) => {
+        btn.addEventListener('click', () => activate(btn.dataset.tab));
+      });
+
+      // connectBridgeEvents wires an SSE stream so the UI can react to live
+      // bridge operations (refreshes, apply actions, and shutdown events).
+      function connectBridgeEvents() {
```

ESLint

[error] Delete `····`

(prettier/prettier)